{"status": "failed", "name": "Matching using a relationship that is already bound, in conjunction with aggregation", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MatchAcceptance2"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486071107, "historyId": "a232d1f2f174d5c8bf644c91b9627ff8", "start": 1578486070891, "steps": [{"status": "passed", "start": 1578486070893, "stop": 1578486071101, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486071101, "stop": 1578486071103, "name": "And having executed", "attachments": [{"source": "0a86eccb-16f0-4c08-a946-7ecd101c4f74-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486071103, "stop": 1578486071105, "name": "When executing query", "attachments": [{"source": "78705351-6588-48b9-a607-0037f4a0b105-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "failed", "attachments": [{"source": "51a5d3dd-0dfa-47eb-bc97-0ae6f316b4be-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486071106, "start": 1578486071105, "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 86, in step_impl\n    assertions.assert_resultset_length(resultset, expected_length)\n  File \"/home/laptop-linux/dev/redis/graph/tests/tck/features/steps/../../utils/assertions.py\", line 195, in assert_resultset_length\n    assert(len(resultset.result_set) == length)\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486071106, "stop": 1578486071107, "name": "And no side effects"}], "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 86, in step_impl\n    assertions.assert_resultset_length(resultset, expected_length)\n  File \"/home/laptop-linux/dev/redis/graph/tests/tck/features/steps/../../utils/assertions.py\", line 195, in assert_resultset_length\n    assert(len(resultset.result_set) == length)\n"}, "uuid": "a31ca0ed-7cce-47ac-b339-a03f5c04eef7"}